docs/guide/acquiring_data/database.rst

```diff
@@ -55,11 +53,13 @@ can also be committed explicitly using the :meth:`Database.commit` method.
         [database]
         url = sqlite:////home/user/sunpy/my_database.sqlite
 
-
-2. Adding new entries
+________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________
+2. <Adding new entries>
 ---------------------
-Each entry in a database is an instance of the class
-:class:`tables.DatabaseEntry` with the following attributes:
+             Each entry in a database is an instance of the class
+             :class:tables.DatabaseEntry (
+                  https://docs.sunpy.org/en/latest/api/sunpy.database.tables.DatabaseEntry.html#sunpy.database.tables.DatabaseEntry
```

Comment:
Adding url of the data base entry file

Reply:
So the code at line 60 here, or the old line 62, already (at least it should) embeds a link to that when the documentation is built. So we don't have to do it manually.